assets/src/js/plugins/give-api/notice.js

```diff
@@ -90,14 +90,22 @@ export default {
 		 * Note: use only in WP Backend
 		 *
 		 * @since 2.5.0
+		 * @since 2.8.0 Localization updated to support changes in WordPress 5.5.
+		 *
 		 * @param {string} notice Notice description.
 		 * @param {string} type   Notice type.
 		 * @param {object} args   Notice type.
 		 *
 		 * @return {string} Notice HTML.
 		 */
 		getAdminNoticeHTML: function( notice, type = 'info', args = { dismissible: true } ) {
-			const btnText = commonL10n.dismiss || '';
+			/**
+			 * WordPress 5.5 removed the localized `commonL10n` in favor of translation in JavaScript.
+			 */
+			const btnText = ( 'undefined' !== typeof commonL10n ) ?
+				commonL10n.dismiss :
+				wp.i18n.__( 'Dismiss this notice.' );
```

Comment:
@kjohnson I will recommend to import `__` from `wp.i18n` and then use `__` in logic.

![image](https://user-images.githubusercontent.com/1784821/91063708-f379c180-e64b-11ea-99ce-4b50d0a5f91d.png)

Reply:
@ravinderk good suggestion. In this case, I'm referencing `wp.i18n` inline behind a conditional tag because it isn't available for this file in WordPress `5.4` - (console error `wp is not defined`).

Reply:
👍